modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java

```diff
@@ -1661,6 +1689,10 @@ else if (active) {
 
 		objectDefinition = objectDefinitionPersistence.update(objectDefinition);
 
+		_objectFolderItemLocalService.updateObjectFolderItem(
+			objectDefinition.getObjectDefinitionId(),
+			objectDefinition.getObjectFolderId(), oldObjectFolderId);
+
```

Comment:
Maybe we should move this code to after the FOR. IDK if Brian will look for sorted order regarding the LocalService.

Reply:
Ok, i will move